R/FilterState.R

```diff
@@ -1664,10 +1664,23 @@ RangeFilterState <- R6::R6Class( # nolint
     # @return numeric(3) with names min, max, step - relevant for sliderInput
     get_pretty_range_inputs = function(values) {
       v_pretty_range <- pretty(values, n = 100)
+      min <- min(v_pretty_range)
+      max <- max(v_pretty_range)
+      range <- max - min
+
+      # ensure number of digits is reasonable
+      # essentially the same as shiny's implementation
```

Comment:
if you think the solution is "essentially the same", then you have to leave a proper reference.